packages/webapp/pages/onboarding.tsx

```diff
@@ -179,13 +190,38 @@ export function OnboardPage(): ReactElement {
     router.replace(getPathnameWithQuery(webappUrl, window.location.search));
   };
 
+  const isFeedSettingsDefined = useMemo(() => !!feedSettings, [feedSettings]);
+
+  const updateSettingsBasedOnExperience = useCallback(() => {
+    if (isSeniorUser && isFeedSettingsDefined) {
+      shouldUpdateAdvancedSettings.current = false;
+      updateAdvancedSettings({
+        advancedSettings: [
+          { id: 5, enabled: false },
+          { id: 10, enabled: false },
+        ],
+      });
+    }
+  }, [isSeniorUser, isFeedSettingsDefined, updateAdvancedSettings]);
+
   const onSuccessfulRegistration = (userRefetched: LoggedUser) => {
     logSignUp({
       experienceLevel: userRefetched?.experienceLevel,
     });
     setActiveScreen(OnboardingStep.EditTag);
   };
 
+  useEffect(() => {
+    if (!shouldUpdateAdvancedSettings.current || !seniorContentOnboarding) {
+      return;
+    }
+    updateSettingsBasedOnExperience();
```

Comment:
Do we even need this update here?
Isn't it enough to simply render with some values and on clicking next we persist whatever the state is?

It should then avoid this useEffect and the whole function basically.

Reply:
What do u mean render with some values? 

There are few points to consider:
1. we've seen scenarios that user can get out of the select tags and whatever onboarding steps that come next meaning if we were to do this somewhere later on we might miss disabling listicles and memes for some devs (of course it wouldn't much impact experiment as we would probably do enrolment only when we can actually disable, but this way we can gain more users to enroll)
2. we can do this when user clicks next on tag selection - probably easy win in this point as we can avoid the use effect, but we'll miss what I mentioned above 
3. we can render the content types with some default values not considering feed settings as default and then update all in the end, but we'll miss what I mentioned above + not a fan of this as the way we are doing things at the moment makes it quite consistent with the feed settings in feed, so wouldn't want to change much.

Reply:
You can still have the enrollment check, but instead of "pre-pushing" it, we can just change the rendering?
Not sure without looking at that code.

The fact if people abort is a very edge-casey thing we should solve that differently regardless I think.

Reply:
> but instead of "pre-pushing" it, we can just change the rendering?

I'm still not sure what u mean by this, u mean the rendering on content types, or?

Reply:
Discussed and decided to move on with removing useEffect and updating the values onClickNext on edit tag page